index.bs

```diff
@@ -3620,8 +3620,9 @@ A few abstract operations are used in this specification for utility purposes. W
 </div>
 
 <emu-alg>
+  1. Assert: _O_ is not *undefined*.
   1. Assert: ! IsPropertyKey(_P_) is *true*.
-  1. If _args_ was not passed, let _args_ be a new empty List.
+  1. Assert: _args_ is a List.
```

Comment:
As I said in https://github.com/whatwg/streams/pull/584#discussion_r85836343 that would defeat the initial point of this PR.